rpiplay.cpp

```diff
@@ -145,6 +154,10 @@ int main(int argc, char *argv[]) {
             low_latency = !low_latency;
         } else if (arg == "-r") {
             rotation = atoi(argv[++i]);
+        } else if (arg == "-w") {
+            display_width = atoi(argv[++i]);
+        } else if (arg == "-h") {
+            display_height = atoi(argv[++i]);
```

Comment:
`-h` is conflicting with the help command

Reply:
Right. I'd prefer a single flag for both width and height, something like -s 1280x720.